packages/charts/examples/components/demo/advanced-usage/legend/legend-metadata/legend-metadata-example.component.ts

```diff
@@ -0,0 +1,157 @@
+    selector: "area-chart-stack-example",
+    templateUrl: "./legend-metadata-example.component.html",
+})
+export class LegendMetadataExampleComponent implements OnInit {
+    public chart: Chart;
+    public chartAssist: ChartAssist;
+    public avgSeries: Partial<IChartSeries<XYAccessors>>
+
+    public ngOnInit() {
+        // areaGrid returns an XYGrid configured for displaying an area chart's axes and other grid elements.
+        this.chart = new Chart(areaGrid());
+        // ChartAssist will use the preprocessor to stack the series' numeric values on the same progression domain
+        this.chartAssist = new ChartAssist(this.chart, stackedArea);
+
+        // Stacked Area accessors let the renderer know how to access x and y domain data respectively from a chart's input data set(s).
+        const accessors = stackedAreaAccessors();
+        // 'y1' defines access to the numeric values we want to visualize.
+        // The items in the data array of this example have a property named 'value',so we'll use that.
+        accessors.data.y1 = (d) => d.value;
+
+        // The area renderer will make the chart look like a area chart.
+        const renderer = new AreaRenderer();
+
+        // In case of a area chart, the scale definitions are flexible.
+        // This example demonstrates a scenario with time on the X scale and a numeric value on the Y scale.
+        const scales: IXYScales = {
+            x: new TimeScale(),
+            y: new LinearScale(),
+        };
+
+        const averageData = calculateAverageSeries(getData());
+        // We are using the base XYRenderer so the metadata does not get displayed on the chart.
+        // Set `ignoreForDomainCalculation` to true to prevent the metadata from affecting the domain.
+        const metaDataRenderer = new XYRenderer({ ignoreForDomainCalculation: true });
+        // Here we create an accessor for our average metadata
+        const avgAccessors = new XYAccessors();
+        // This is so the legend knows the value for the y
+        avgAccessors.data.y = (d) => d.value;
+
+        this.avgSeries = {
+            ...averageData,
+            accessors: avgAccessors,
+            renderer: metaDataRenderer,
+            scales: scales,
+            // showInLegend is false because we manually add our own series
+            showInLegend: false,
+        }
+        // Here we assemble the complete chart series.
+        let seriesSet: Partial<IChartSeries<IAccessors>>[] = getData().map(d => ({
+            ...d,
+            accessors,
+            renderer,
+            scales,
+        }));
+        // Combining both sets in an array of series
+        seriesSet = [...seriesSet, this.avgSeries];
+
+        // Finally, pass the series set to the chart's update method
+        this.chartAssist.update(seriesSet as IChartSeries<IAreaAccessors>[]);
+    }
+}
+
+function calculateAverageSeries(dataArr: Partial<IDataSeries<XYAccessors>>[]): Partial<IDataSeries<XYAccessors>> {
```

Comment:
We've been using `seriesSet` as the symbol name when we're referring to multiple series. Do you mind changing it:

```suggestion
function calculateAverageSeries(seriesSet: Partial<IDataSeries<XYAccessors>>[]): Partial<IDataSeries<XYAccessors>> {
```